e2e/tests/localAuthoritySubmitsApplication_test.js

```diff
@@ -9,12 +10,20 @@ const otherProceedings = require('../fixtures/otherProceedingData');
 const ordersAndDirectionsNeeded = require('../fixtures/ordersAndDirectionsNeeded.js');
 
 let caseId;
+let nonReuseCaseId;
 
 Feature('Local authority creates application');
 
-async function setupScenario(I) {
-  if (!caseId) { caseId = await I.submitNewCase(config.swanseaLocalAuthorityUserOne); }
-  await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+async function setupScenario(I, reuse = true) {
+  if (reuse == false) {
```

Comment:
Could check !reuse?

Also could maybe simplify to check reuse and use that to set a caseId variable and then pass this to the submitNewCase and navigateToCaseDetailsAs functions to reduce code duplication.